library/cc/stream.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include <cstddef>
+#include <vector>
+
+#include "library/common/types/c_types.h"
+#include "request_headers.h"
+#include "request_trailers.h"
+
+class Stream {
+public:
+  Stream(envoy_stream_t handle);
+
+  Stream& send_headers(const RequestHeaders& headers, bool end_stream);
+  Stream& send_data(const std::vector<std::byte>& data);
```

Comment:
Is there a pybind reason to use this buffer representation? If not, I'd propose we use uint8_t (which Envoy currently uses internally) and probably just simple arrays.

Reply:
there is not, i'll use uint8_t.